torch/distributed/pipelining/schedules.py

```diff
@@ -2070,6 +2141,331 @@ def backward_stage_index(step):
             enable_zero_bubble=True,
         )
 
+
+class ScheduleZBVZeroBubble(PipelineScheduleMulti):
+    """
+    The ZBV Zero Bubble schedule.
```

Comment:
Can you include a short description of the differences between ZBV and the existing InterleavedZeroBubble schedule?